website/docs.md

```diff
@@ -721,7 +727,7 @@ id. Then comes the file and line where the log was produced, followed finally by
 the contents of the log.
 
 The main component returned by `weaver.Init` has a Logger method as well (like
-all `weaver.Instance`s):
+all `weaver.Instances`):
```

Comment:
I'd revert this as `weaver.Instance` is a thing.

Reply:
It looks ugly, hence I changed it. It displays as weaver.Instance s. It's not a big deal though.